src/Microsoft.AspNet.Mvc.ViewFeatures/Rendering/StringCollectionTextWriter.cs

```diff
@@ -28,7 +29,7 @@ public class StringCollectionTextWriter : TextWriter
         public StringCollectionTextWriter(Encoding encoding)
         {
             _encoding = encoding;
-            Buffer = new BufferEntryCollection();
+            Content = new BufferedHtmlContent();
```

Comment:
Not sure about the need for changes to this class. It's a `TextWriter` and as such it can safely just operate on strings.